src/components/VideoCard/VideoCard.component.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useRef, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { Paper } from '@material-ui/core';
+
+import {
+  VideoThumbnail,
+  VideoInfo,
+  VideoTitle,
+  VideoDescription,
+} from './VideoCard.module';
+
+import './VideoCard.styles.css';
+
+function HomeView(props) {
+  const { title, thumbnail, description, videoId } = props;
+  const [isCardHovered, setIsCardHovered] = useState(false);
+  const cardRef = useRef(null);
+
+  const handleMouseOver = () => {
+    setIsCardHovered(true);
+  };
+
+  const handleMouseOut = () => {
+    setIsCardHovered(false);
+  };
+
+  useEffect(() => {
+    const node = cardRef.current;
+    if (node) {
+      node.addEventListener('mouseover', handleMouseOver);
+      node.addEventListener('mouseout', handleMouseOut);
+      return () => {
+        node.removeEventListener('mouseover', handleMouseOver);
+        node.removeEventListener('mouseout', handleMouseOut);
+      };
+    }
```

Comment:
This is not needed whatsoever, you can achieve the same result by making use of css pseudo-selectors like `:hover`